.github/workflows/ProductionRelease.yml

```diff
@@ -0,0 +1,68 @@
+name: Create Release
+
+on:
+  push:
+    branches: [ "prod" ]
+
+jobs:
+  Build_Release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Master
+      uses: actions/checkout@v3
+      with:
+        submodules: true
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v2
+      with:
+        dotnet-version: 6.0.x
+    - name: Restore dependencies
+      run: dotnet restore
+    - name: Build
+      run: dotnet build --no-restore
+    - name: Test Build
+      run: dotnet test --no-build --verbosity normal
+    - name: Setup build artifacts
+      run: |
+        mkdir Artifacts
+        mv ValheimExportHelper/bin/Debug/net6.0/ValheimExportHelper.dll  Artifacts/ValheimExportHelper.dll
+        mv ValheimExportHelper/bin/Debug/net6.0/YamlDotNet.dll  Artifacts/YamlDotNet.dll 
+    - name: Upload a Build Artifact
+      uses: actions/upload-artifact@v3.1.0
+      with:
+    # Artifact name
+        name: 'ValheimExportHelper.dll'
+    # A file, directory or wildcard pattern that describes what to upload
+        path: 'Artifacts/' 
+    
+    - name: Grab Version
+      run: |
+        cat ValheimExportHelper/ValheimExportHelper.csproj | grep '<Version>' > tmp.txt
+        sed 's|[<Version>,]||g' tmp.txt | sed 's|[/,]||g' > VERSION
+      
+    - name: Commit Version
+      uses: EndBug/add-and-commit@v7
+      with:
+        author_name: "Bryan Smith"
+        author_email: "sbtoonz@gmail.com"
+        message: "Update Version"
+        add: "VERSION"
```

Comment:
What does this do?

Reply:
> What does this do? (or rather what's its purpose)

It should be redone to a generic author but it commits the Version tag from the SLN -> VERSION file to extract for the Tag/Name

Reply:
Is it not obtainable without a commit?